src/olympia/urls.py

```diff
@@ -75,6 +75,8 @@
 
     # Javascript translations.
     # Should always be called with a cache-busting querystring.
+    # Still served for the moment, but superseded by a fully static version we
+    # generate at deploy-time through our generate_jsi18n_files command.
```

Comment:
mmm, what's the logic behind not just deleting it in one go?  The templates have been changed (above) to point to the new location so what would be using it the old dynamic url?

Reply:
I was worried about old pages cached in proxies/browser caches mostly. Not a huge deal admittedly, but I'm not sure what's out there...